src/OpenTelemetry/Logs/LogRecord.cs

```diff
@@ -80,6 +81,15 @@ internal LogRecord()
         }
     }
 
+    internal enum LogRecordSource
+    {
+#pragma warning disable SA1602 // Enumeration items should be documented
```

Comment:
Is a pragma disable _really_ necessary here? 😄

Reply:
Opened this PR to loosen the requirements: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5256

Reply:
Ended up adding docs. Not sure if that PR will land.